cns/imds/client_test.go

```diff
@@ -111,7 +111,7 @@ func TestGetNetworkInterfaces(t *testing.T) {
             },
             {
                 "interfaceCompartmentID": "",
-                "macAddress": "00:00:5e:00:53:02"
+                "macAddress": "00005e005302"
```

Comment:
should we add a new test case instead of changing the existing one to ensure there are no regressions with the existing behavior?

Reply:
Actually, this test and the fix for it are for the same vnet scale Swift v2 scenario. This is the first time CNS queries to IMDS to get NIC and MAC from it, also happens to have Windows MAC address format so we don't have an existing behavior to regress.